doc/whats-new.rst

```diff
@@ -31,59 +31,38 @@ What's New
 v0.10.4 (unreleased)
 --------------------
 
+Documentation
+~~~~~~~~~~~~~
+
 Enhancements
 ~~~~~~~~~~~~
 
-- Support writing lists of strings as netCDF attributes (:issue:`2044`).
```

Comment:
please fix -- you are inadvertently editing release notes for 0.10.3.

Reply:
Sorry. Fixed it.

Reply:
Fixed.